README.md

```diff
@@ -24,3 +24,14 @@ HTCondor-CE RPMs are available from the following locations:
 
 - HTCondor Yum repositories: https://research.cs.wisc.edu/htcondor/yum/
 - OSG Yum repositories: https://opensciencegrid.org/docs/common/yum/
+
+Versioning
+----------
+
+HTCondor-CE follows the same stable/development release series model as [HTCondor](https://htcondor.readthedocs.io/en/latest/version-history/introduction-version-history.html).
```

Comment:
```suggestion
HTCondor-CE follows the same even stable/odd development release series model as [HTCondor](https://htcondor.readthedocs.io/en/latest/version-history/introduction-version-history.html).
```
That way most people won't have to follow the link to understand the sentence.

Reply:
That isn't the case, though? 3 is stable, 4 is devel

Reply:
So it's the opposite of the Condor versioning scheme (which explicitly says "Even numbers represent stable versions, while odd numbers represent development versions.").
Maybe just drop that sentence and rephrase the paragraph as follows:
```
As of September 2019, we maintain two different versions of HTCondor-CE:

- HTCondor-CE 4 ([documentation](https://htcondor-ce.readthedocs.io/en/latest/)),
  kept in the [master](https://github.com/htcondor/htcondor-ce/tree/master) branch
- HTCondor-CE 3 ([documentation](https://htcondor-ce.readthedocs.io/en/stable/)),
  kept in the [stable](https://github.com/htcondor/htcondor-ce/tree/stable) branch
```